src/main/java/com/akathist/maven/plugins/launch4j/Launch4jMojo.java

```diff
@@ -456,6 +462,36 @@ private void setPermissions(File workdir) {
     }
 
     /**
+     * If custom header objects or libraries shall be linked, they need to sit inside the launch4j working dir.
```

Comment:
That seems to be the way how it is implemented in launch4j. It will try to construct an absolute path using `new File(_basedir, pathToMyHeader)` which results in "C:\..\.m2\..\workdir\C:\..\projectdir". Now, when I use your plugin I don't wanna make assumptions on where that working dir may reside and try building relative paths.

Let me know if there is a better way to acheive this.